### PR TITLE
missing tags added

### LINKED ISSUE #255: New Eclipse warnings
Following https://github.com/pytrip/pytrip/issues/193#issuecomment-281784900

### PR DESCRIPTION
**Pull request by grzanka**
Fixes #255

### CODE REVIEW

**Comment by grzanka** [comment]
Seems to work:

```
→ dcmdump 2/RTSTRUCT.PYTRIP.dcm| grep StudyID
(0020,0010) SH [1]                                      #   2, 1 StudyID
→ dcmdump 2/RTSTRUCT.PYTRIP.dcm| grep Referr 
(0008,0090) PN [py^trip]                                #   8, 1 ReferringPhysicianName
→ dcmdump 2/RTSTRUCT.PYTRIP.dcm| grep SeriesNumber
(0020,0011) IS [1]                                      #   2, 1 SeriesNumber
→ dcmdump 2/RTSTRUCT.PYTRIP.dcm| grep Manufacturer
(0008,0070) LO [  InitPatientPosition: HFS; PatientOrientation: L/P; BodyPart: HEA... # 162, 1 Manufacturer
→ dcmdump 2/RTSTRUCT.PYTRIP.dcm| grep ROIG        
(3006,0036) CS [0]                                      #   2, 1 ROIGenerationAlgorithm
```

**Comment by grzanka** [comment]
@okejdokej I do not see your comments

File: pytrip/vdx.py [hunk 1]
```diff
@@ -267,8 +267,11 @@ def create_dicom(self):
...
         if self.cube is not None:
             ds.PatientsName = self.cube.patient_name
+            ds.Manufacturer = self.cube.creation_info  # Manufacturer tag, 0x0008,0x0070 (type LO - Long String)
         else:
-            ds.PatientsName = ""
+            ds.PatientsName = ''
+            ds.Manufacturer = ''  # Manufacturer tag, 0x0008,0x0070 (type LO - Long String)
+        ds.SeriesNumber = '1'  # SeriesNumber tag 0x0020,0x0011 (type IS - Integer String)
         ds.PatientID = "123456"
         ds.PatientsSex = '0'
```
Comment (okejdokej) [inline pytrip/vdx.py]: PatientsSex: tag (0010,0040); Description: Sex of the named patient. Enumerated Values: M = male F = female O = other.
I think that '0' should be changed to O (other) .

File: pytrip/dos.py [hunk 2]
```diff
@@ -204,7 +204,7 @@ def create_dicom(self):
         ds.NumberOfFrames = len(self.cube)
         ds.PixelRepresentation = 0
         ds.StudyID = '1'
-        ds.SeriesNumber = 14
+        ds.SeriesNumber = '14'  # SeriesNumber tag 0x0020,0x0011 (type IS - Integer String)
```
Comment (okejdokej) [inline pytrip/dos.py]: Just curiosity, why 14? why not '1' instead?

File: pytrip/cube.py [hunk 3]
```diff
@@ -481,8 +481,8 @@ def create_dicom_base(self):
         ds.PixelSpacing = [self.pixel_size, self.pixel_size]
 
         # Add eclipse friendly IDs
-        ds.StudyID = ''  # Study ID tag 0x0020,0x0010
-        ds.ReferringPhysiciansName = ''  # Referring Physician's Name tag 0x0008,0x0090
+        ds.StudyID = '1'  # Study ID tag 0x0020,0x0010 (type SH - Short String)
```
Comment (okejdokej) [inline pytrip/cube.py]: This value can be also a string, maybe 'PYTRIP'? However, '1' is ok.

File: pytrip/vdx.py (hunk 1 above)
Comment (nbassler) [inline pytrip/vdx.py]: Will be fixed with PR #258 via commit https://github.com/pytrip/pytrip/pull/258/commits/950db42f4a3a8dd62704cf441ebfb51fffce9066

File: pytrip/dos.py (hunk 2 above)
Comment (nbassler) [inline pytrip/dos.py]: I guess simply not to have '1' everywhere... But not sure.

File: pytrip/cube.py (hunk 3 above)
Comment (nbassler) [inline pytrip/cube.py]: ok, good to know.

**Pull request merged**